pkg/apis/proto/daemon/daemon.proto

```diff
@@ -14,72 +14,74 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-syntax = "proto2";
+syntax = "proto3";
+
 option go_package = "github.com/numaproj/numaflow/pkg/apis/proto/daemon";
 
 import "google/api/annotations.proto";
 import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
 
 package daemon;
 
 // BufferInfo is used to provide information about the buffer including
 // pending messages count, if it has unprocessed data etc which helps in pipeline/buffer deletion etc
 message BufferInfo {
-  required string pipeline = 1;
-  required string bufferName = 2;
-  required int64 pendingCount = 3;
-  required int64 ackPendingCount = 4;
+  string pipeline = 1;
+  string bufferName = 2;
+  google.protobuf.Int64Value pendingCount = 3;
+  google.protobuf.Int64Value ackPendingCount = 4;
   // Total messages existing in the buffer, including pending, ackPending and acked.
-  required int64 totalMessages = 5;
-  required int64 bufferLength = 6;
-  required double bufferUsageLimit = 7;
-  required double bufferUsage = 8;
-  required bool isFull = 9;
+  google.protobuf.Int64Value totalMessages = 5;
```

Comment:
In proto3, `int64` and `uint64` data will be serialized as `string` when converting to JSON, which makes trouble for unmarshaling to a struct. Use `google.protobuf.Int64Value` to solve the problem.

Reply:
done.